pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1079,10 +1079,10 @@ func (ctrl *PersistentVolumeController) deleteVolumeOperation(arg interface{}) e
 		} else {
 			// The plugin failed, mark the volume as Failed and send Warning
 			// event
-			if _, err = ctrl.updateVolumePhaseWithEvent(volume, v1.VolumeFailed, v1.EventTypeWarning, "VolumeFailedDelete", err.Error()); err != nil {
-				glog.V(4).Infof("deleteVolumeOperation [%s]: failed to mark volume as failed: %v", volume.Name, err)
+			if _, err2 := ctrl.updateVolumePhaseWithEvent(volume, v1.VolumeFailed, v1.EventTypeWarning, "VolumeFailedDelete", err.Error()); err2 != nil {
```

Comment:
I think you can just add a colon, no need to create err2 https://play.golang.org/p/tgrIrB8hmZ